switchbot_ros/requirements.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
This is my just curiosity.
Why is empty requirements.txt recommended? For future developers or catkin_virtualenv requirements?

Reply:
I thought there is no `requirements.txt`, the catkin_virtualenv fails to create python environment, but it seems it doesn't need to.

Reply:
Thank you. I see.